.github/workflows/ci.yml

```diff
@@ -145,7 +147,7 @@ jobs:
           key: yarn-${{ hashFiles('yarn.lock') }}
       - name: Install
         run: |
-          BABEL_ENV=test-legacy make -j bootstrap-only
```

Comment:
We don't need to run this anymore?

Reply:
Yeah. The `BABEL_ENV=test-legacy` impacts how Babel is built. But here we are basically running `yarn install` so it is an no-op. I suggest we simply put `yarn install` here.